waku/nwaku.go

```diff
@@ -496,6 +495,8 @@ func newWakuNode(ctx context.Context, config *WakuConfig) (*WakuNode, error) {
 
 	wg.Add(1)
 	n.wakuCtx = C.cGoWakuNew(cJsonConfig, resp)
+	n.MsgChan = make(chan common.Envelope, 100)
```

Comment:
Chan chan, chan chan chan :)
```suggestion
	n.MsgChan = make(chan common.Envelope, MaxMessageChanWakuChanChanChan)
```